### PR TITLE
Fix is-link style to take colors from wp-admin theme.

### PR DESCRIPTION
**Pull request by jasmussen**
## Description

`is-link` buttons did not get their colors from your wp-admin color theme. This PR fixes that. Before:

Screenshot 2021-04-14 at 08 37 05

After:

after

Previously: https://github.com/WordPress/gutenberg/pull/6562

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] I've tested my changes with keyboard and screen readers. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal).

### CODE REVIEW

File: packages/components/src/button/style.scss [hunk 1]
```diff
@@ -210,26 +210,16 @@
...
-			box-shadow: none;
-		}
-
 		&:focus {
-			color: #124964;
-			box-shadow:
-				0 0 0 $border-width #5b9dd9,
-				0 0 var(--wp-admin-border-width-focus) $border-width rgba(30, 140, 190, 0.8);
+			border-radius: $radius-block-ui;
 		}
```
Comment (youknowriad) [inline packages/components/src/button/style.scss:223]: Why did you remove all these styles? I worry about the impact

Comment (jasmussen) [reply packages/components/src/button/style.scss:223]: Because those styles are from the older wp-admin styles. I don't mean to dismiss the feedback, but it's similar to when the button styles changed from bevelled to flat.

**Review by youknowriad — APPROVED** [review]
LGTM
